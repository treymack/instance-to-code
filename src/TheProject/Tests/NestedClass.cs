using System;
using System.Linq;
using ApprovalTests;
using NUnit.Framework;

namespace TheProject.Tests
{
    [TestFixture]
    public class NestedClass
    {
        [Test]
        public void CanScriptToCode()
        {
            var instance = new NestedClassTestCase();

            var code = ITC.Instance.ToCode(instance);

            Approvals.Verify(code);
        }

        public class NestedClassTestCase
        {
        }
    }
}
