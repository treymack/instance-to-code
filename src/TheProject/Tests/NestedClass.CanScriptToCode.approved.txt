new TheProject.Tests.NestedClass.NestedClassTestCase()
{
}